includes/class-wc-stripe-webhook-handler.php

```diff
@@ -1090,7 +1090,7 @@ public function process_payment_intent( $notification ) {
 		// Set the order being processed for the `wc_stripe_webhook_received` action later.
 		$this->resolved_order = $order;
 
-		if ( $this->lock_order_payment( $order, $intent ) ) {
+		if ( WC_Stripe_Order_Helper::lock_order_payment( $order, $intent ) ) {
```

Comment:
This function no longer seems to accept the `$intent` parameter

```suggestion
		if ( WC_Stripe_Order_Helper::lock_order_payment( $order) ) {
```

Reply:
Correct! I have updated them in my latest commits